DEVELOPMENT.md

```diff
@@ -0,0 +1,91 @@
+# Development
+
+This doc explains how to setup a development environment so you can get started
+contributing to `Knative-GCP`.
+Also take a look at:
+
+- [The pull request workflow](https://www.knative.dev/contributing/contributing/#pull-requests)
+- [How to add and run e2e tests](./test/e2e/README.md)
+- [Iterating](#iterating)
+- [Format check](#format-check)
+
+## Getting started
```

Comment:
I think we might need: https://github.com/knative/eventing/blob/master/DEVELOPMENT.md#requirements
and also https://github.com/knative/eventing/blob/master/DEVELOPMENT.md#setup-your-environment here

Reply:
This part is included in https://github.com/google/knative-gcp#installing-knative-gcp which is linked below in this doc